README.md

```diff
@@ -1,81 +1,323 @@
+2. Copy [.env.example](./.env.example) file to ./.env file).
+    ```shell
+    cp .env.example .env
+    ```
+3. Configure API endpoints
+    
+    **Kultus API**
+    ```shell
+    NEXT_PUBLIC_API_BASE_URL=https://kultus.api.test.hel.ninja/graphql
+    # for local development, use the following line instead
+    # NEXT_PUBLIC_API_BASE_URL=http://localhost:8081/graphql
+    ```
+4. Optional. If you set up a local API, remember to set up also a local LinkedEvents. You can find the instructions for that from Kultus API documentation https://github.com/City-of-Helsinki/palvelutarjotin. To find out why's that important, see [the data model instructions](#about-kultus-data-models-and-relations-to-linkedevents).
 
-1. Set up palvelutarjotin API locally, see [palvelutarjotin README](https://github.com/City-of-Helsinki/palvelutarjotin/blob/master/README.md)
-2. Copy `.env.local.example` to `.env.local` and fill/update the required values
-3. Run `yarn` to install dependencies
-4. Run `yarn dev` to start the development server
+    > NOTE: if you have any problems to connect, double check the network port where the API is running. Running Docker services and their port forwardings can be checked with `docker ps -a`. 
+    >
+    > If you connect to a Docker container from your local computer, you can usually use https://localhost:8081/ to reach the API. 
 
-## Credentials
+5. Run the Node application in development mode with `yarn dev`.
 
-### Google reCAPTCHA
+##### Running using Docker
 
-Palvelutarjotin-ui uses googles reCAPTCHA v3:
+Using the following instructions you should be able to:
+
+- Run this UI using Docker & Docker compose
+- Run the Kultus backend locally in Docker or use the public test environment's backend
+
+1. Copy [.env.example](./.env.example) file to ./.env file).
+    ```shell
+    cp .env.example .env
+    ```
+2. Configure API endpoints
+    
+    **Kultus API** 
+    ```shell
+    NEXT_PUBLIC_API_BASE_URL=https://kultus.api.test.hel.ninja/graphql
+    ```
+    to use the API from the public test environment.
+    > NOTE: If you connect to a Docker container from your local computer, you can usually use https://localhost:8081/ to reach the API, but if you connect from a Docker container to another, you should be using the container names like http://kultus-backend:8081/ to reach the API or use http://host.docker.internal:8081/ to connect via your local machine ports.
+
+
+3. To build and start a docker container, you can use
+    **a development build:**
+    > NOTE: [compose.yml](./compose.yml) maps some local directories as a volume in order to get a hot reload for the code to work.
+    ```shell
+    DOCKER_TARGET=development docker compose up --build
+    ```
+
+    or 
+
+    if you want to run **a production build**, you most likely first need to unmap the composer volumes from [compose.yml](./compose.yml), by removing or fully commenting out the following section:
```

Comment:
Same thing here as in the palvelutarjotin-admin PR, using compose.yml & compose.override.yml this could possibly be handled without having to make changes to compose.yml file which is committed to the repository. So, an idea for possible further development, not necessary to do in this PR.

Reply:
> I would be happy to remove the develop stage from dockerfile, remove the volumes mapping totally and make the production build stage the default stage.

https://github.com/City-of-Helsinki/palvelutarjotin-admin/pull/401#discussion_r2065889961

Reply:
> > I would be happy to remove the develop stage from dockerfile, remove the volumes mapping totally and make the production build stage the default stage.
> 
> [City-of-Helsinki/palvelutarjotin-admin#401 (comment)](https://github.com/City-of-Helsinki/palvelutarjotin-admin/pull/401#discussion_r2065889961)

Let's discuss this live.